package-lock.json

```diff
@@ -10,9 +10,9 @@
       "integrity": "sha512-sW77BFwJ48YvQp3Gzz5xtAUiXuYOL2aMJKDwiaY3OcvdqBFurtYfOpSa4QrNyDxmOGRFSYzUpabU2m9QrlWE7w==",
       "dev": true,
       "requires": {
-        "chalk": "2.3.0",
```

Comment:
is all this needed?

Reply:
Some could potentially be cleaned up, but they're dev dependencies. Chalk is required by `postcss`, which is being used for auto-prefixing